stable/grafana/templates/_pod.tpl

```diff
@@ -253,6 +253,11 @@ containers:
         value: "{{ $value }}"
 {{- end }}
     {{- if .Values.envFromSecret }}
+    envFrom:
+      - secretRef:
+          name: {{ .Values.envFromSecret }}
+    {{- end }}
+    {{- if .Values.envRenderSecret }}
     envFrom:
       - secretRef:
```

Comment:
If `envFromSecret` and `envRenderSecret` are both set will we have two `envFrom` sections. Is that intended/desirable?

Reply:
If both keys are set, there will be two `envFrom` sections. I believe users should decide which env variables to place under each key (based on convenience). I also believe that only one key either `envFromSecret` or `envRenderSecret` should be used at a time. However, chart does not prohibit the usage of both keys and it's a user's responsibility to handle such cases.